documentation/user_docs/docs/conf.py

```diff
@@ -30,7 +30,12 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
+    "sphinxcontrib.matlab",
```

Comment:
Do any of these make external dependencies we need to worry about in the build process?

Reply:
Yeah, I added a `pip install sphinxcontrib-matlabdomain` to the `Jenkinsfile`... 
I don't know if we want to use this though... what do you think of the generated output, e.g. https://mducle.github.io/horace_docs_unstable/API_Reference.html#sqw.@SQWDnDBase.disp2sqw_eval ?